Already defined by main/musl/loongarch64-add-fpu-and-simd-context-to-signal.h.patch

--- a/src/runtime/loongarch64-linux-os.c
+++ b/src/runtime/loongarch64-linux-os.c
@@ -42,7 +42,6 @@
 #include <stdint.h>
 #include <stddef.h>
 #include <sys/ucontext.h>   /* for ucontext_t / mcontext_t */
-#include <asm/sigcontext.h> /* for sctx_info, FPU_CTX_MAGIC, struct fpu_context */
 
 
 #define EXTCTX_SCAN_MAX_BYTES 16384
